index.js

```diff
@@ -60,7 +60,7 @@ function simpleGet (opts, cb) {
       else return simpleGet(opts, cb)
     }
 
-    const tryUnzip = typeof decompressResponse === 'function' && opts.method !== 'HEAD'
+    const tryUnzip = !opts.noDecompress && typeof decompressResponse === 'function' && opts.method !== 'HEAD'
```

Comment:
Rename the option to `decompress` and change the check to `decompress === false`